src/selectize/choices.tpl.html

```diff
@@ -1,5 +1,4 @@
-<div ng-show="$select.open" class="ui-select-choices ui-select-dropdown selectize-dropdown"
```

Comment:
I`ve moved the dropdown classes and functionality to here https://github.com/angular-ui/ui-select/pull/1930/files/fdae8530b84f3426b3b411da2cf840248a09155a#diff-55a8341ed55143ae563b5f1e54237ca8R13